lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2779,9 +2772,41 @@ bool TypeSystemSwiftTypeRef::DumpTypeValue(
           s, format, data, data_offset, data_byte_size, bitfield_bit_size,
           bitfield_bit_offset, exe_scope, is_base_class);
     }
-    case Node::Kind::Structure:
     case Node::Kind::BoundGenericStructure:
       return false;
+    case Node::Kind::Structure: {
+      // In some instances, a swift `structure` wraps an objc enum. The enum
```

Comment:
do we know what "some instances" are?

Reply:
There are many instances. I'm not currently sure which objc enums are mangled as structs, and which aren't, nor why. It's something I could look into.

Reply:
Let's save that investigation for another day.